ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -24,6 +28,10 @@ images:
   dockerfile_path: Dockerfile
   from: base
   to: dpdk
+- context_dir: .
+  dockerfile_path: ztp/resource-generator/Containerfile
```

Comment:
IMHO it's bad idea, calling for trouble
but have it whatever way you want.

Reply:
Upstream cnf-features-deploy already has ztp/resource-generator/Containerfile, and has since its infancy (which was honestly not that long ago :)

Until / unless we rename it there, this has to match :)

Reply:
or be fixed there..